frontend/ts-src/conf.ts

```diff
@@ -1,10 +0,0 @@
-export type SentryConf = { dsn: string }
-
-export class Conf {
-    static get(): Conf {
-        return new Conf({dsn: 'https://268b122ecafb4f20b6316b87246e509c@o937148.ingest.sentry.io/5887547'})
```

Comment:
should we remove the frontend sentry sdk completely? or should we set it up differently?